app/models/interest.rb

```diff
@@ -1,3 +1,5 @@
 class Interest < ApplicationRecord
-  belongs_to :user
+  self.inheritance_column = :_type_disabled
+  enum type: { health: 0, hobby: 1, work: 2 }
+  belongs_to :user, optional: true
```

Comment:
@kovson 
Ok but when i want to create User with a interest that give me rollback an user can't create without that  logs : https://gist.github.com/synion/6110d17add32608a071a939607392b52